bin/plugin/restricted/accountListAccesses

```diff
@@ -42,23 +50,21 @@ $fnret = OVH::Bastion::is_bastion_account_valid_and_existing(account => $account
 $fnret or osh_exit $fnret;
 $account = $fnret->value->{'account'};
 
-# FIXME won't be able to see group accesses caller is not a member of
-my $accessListResult = OVH::Bastion::get_acls(account => $account);
-$accessListResult or osh_exit $accessListResult;
+$fnret = OVH::Bastion::get_acls(account => $account);
+$fnret or osh_exit $fnret;
 
-if (not @{$accessListResult->value}) {
-    osh_ok R('OK_EMPTY', msg => "This account has no registered accesses to machines through this bastion yet");
-}
-else {
-    osh_info "NOTE: KNOWN LIMITATION in this version: you won't be able to";
-    osh_info "see access lists of group you're not yourself a member of.";
-    osh_info "For such cases, an error 'unable to access' will be printed.";
-    osh_info ' ';
-    osh_info "This account has access to the following servers:";
+if (!OVH::Bastion::is_auditor(account => $self)) {
+    osh_info "NOTE: you're not a bastion auditor, hence you won't be able to";
+    osh_info "see access lists of groups you're not yourself a member of.";
     osh_info ' ';
+}
 
-    OVH::Bastion::print_acls(acls => $accessListResult->value, reverse => $reverse, hideGroups => $hideGroups);
-    osh_ok($accessListResult);
+if (not @{$fnret->value}) {
+    osh_ok R('OK_EMPTY', msg => "This account has no registered accesses to machines through this bastion yet");
 }
```

Comment:
Missing return or else ?

Reply:
`osh_ok` is an `exit` ;) this makes a cleaner code and closer to the sibling command `selfListAcceesses`